PandaGrom/hometask-3/run.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# This module asks the user to enter data
+module RunCli
+  def run_cli
+    loop do
+      input_town = users_input_town
+
+      break if input_town == 'exit!'
+
+      input_year = users_input_year
+
+      if input_town.empty?
+        error
+      else
+        input_year == '2020' ? show(input_town, LAST_YEAR) : show(input_town, THE_YEAR_BEFORE_LAST)
```

Comment:
why do you specify the year?
I suppose you do not have to do it according to the task

Reply:
if you decided to do it like that, you can add some info about available years to your console output
because it's not clear for the users which options can they input (without looking at the code)